python/cuda_cccl/pyproject.toml

```diff
@@ -19,11 +19,12 @@ requires-python = ">=3.9"
 dependencies = [
   "numba>=0.60.0",
   "numpy",
-  "cuda-python==12.9.0",
+  "cuda-bindings>=12.9.1,<13.0.0",
   "cuda-core",
   "nvidia-cuda-nvrtc-cu12",
   "nvidia-nvjitlink-cu12",
   "pynvjitlink-cu12>=0.2.4",
+  "numba-cuda",
```

Comment:
Are you able to set a lower bound here? Adding a lower bound here would help ensure you have full support for the new cuda-bindings code that no longer uses pynvjitlink. I think `>=0.16.0` might be enough for that, but it's just as good to use the latest minor `>=0.18.0` if you're willing to take a higher lower bound. RAPIDS is switching to `>=0.18.0`.